martin/src/mbtiles/mod.rs

```diff
@@ -149,8 +203,10 @@ mod tests {
                 pm-src3: https://example.org/file3.ext
                 pm-src4:
                   path: https://example.org/file4.ext
+            validate: thorough
```

Comment:
minor nit: probably best to put the simple `key: value` things before the nested ones - i.e. move these two above `paths:`